### ISSUE TITLE
Clean up Namibia room spray validation rules + fix text formatting

### ISSUE THREAD

**Issue by sbabur** [disarm-platform/user-requests-and-feedback#79]
1) Make I vs. i consistent, remove space between ( and I
2) Update below data validation text to say: "Number of rooms sprayed with DDT + Deltamethrin + Actellic must be <= total number of rooms sprayed"

![image](https://user-images.githubusercontent.com/37877007/42514609-196f2656-845a-11e8-902a-d27f4f08cead.png)

![image](https://user-images.githubusercontent.com/37877007/42514525-eb7d0a9c-8459-11e8-88e6-c8d6a9b0b553.png)

**Comment by onlyjsmith** [comment]
@sbabur done on staging - please validate

**Comment by sbabur** [comment]
@onlyjsmith Can you rephrase the warning to say:  
"Total number sprayed (ddt + deltamethrin + actellic) should be less than or equal to the total number of sprayable rooms"

I got the field language wrong - it's # of sprayable rooms, not # sprayed that should be <= ddt+delta+actellic

![image](https://user-images.githubusercontent.com/37877007/42677673-bfa4b9a4-867c-11e8-8fc5-74941ed88a08.png)

**Comment by sbabur** [comment]
@onlyjsmith Other than the language - all good

**Comment by onlyjsmith** [comment]
@sbabur Thanks. Think would be good to clarify. I'll post the exact validation config below, we can either go through it next week, or if you can figure out what it does, feel free to post an updated version!

```json
[
	{
	  "name": "sprayed more than total structures",
	  "expression": "numbersprayed_ddt + numbersprayed_delta + numbersprayed_actellic <= number_sprayable",
	  "message": "Total sprayed should less than or equal to total number sprayed (ddt + deltamethrin + actellic)",
	  "type": "warning"
	}, {
	  "name": "sprayed more than total structures",
	  "expression": "numbersprayed_ddt + numbersprayed_delta + numbersprayed_actellic + number_unsprayed == number_sprayable",
	  "message": "Total should equal number sprayed with ddt + deltamethrin + actellic + sprayable-but-unsprayed",
	  "type": "warning"
	}, {
	  "name": "population",
	  "expression": " total_population_sprayedrooms <= house_population",
	  "message": "More people sleeping in sprayed rooms than population of houshold.",
	  "type": "warning"
	}
]
```

**Comment by onlyjsmith** [comment]
@sbabur maybe simpler is our validations tester as part of the Debug tools: https://nam.app-stage.disarm.io/debug/validations

It's a little fiddly, and won't let you edit them directly - you must click 'export validations' and send us the file if you make any changes.

But there's a tester built-in, and you can edit the expressions and then test them directly. Thoko or I can give you some more pointers on it next week if useful.

![image](https://user-images.githubusercontent.com/148280/42696125-8f2f0596-86b7-11e8-81d8-39ab289eea3c.png)

![image](https://user-images.githubusercontent.com/148280/42696182-b68dd11c-86b7-11e8-9fa5-11f6c9a8663d.png)

**Comment by sbabur** [comment]
This should suffice for math checks:

[
	{
	  "name": "sprayed more than total structures",
	  "expression": "numbersprayed_ddt + numbersprayed_delta + numbersprayed_actellic <= number_sprayable",
	  "message": "Total number sprayed (ddt + deltamethrin + actellic)  should be less than or equal to the total number of sprayable rooms",
	  "type": "warning"
	}, {
	  "name": "population",
	  "expression": " total_population_sprayedrooms <= house_population",
	  "message": "More people sleeping in sprayed rooms than population of houshold.",
	  "type": "warning"
	}
]

Iitula asked today if we could check that Household health 1 == Household Health 2. Is it possible to compare that an alphanumeric string matches what was entered in another field?

**Comment by sbabur** [comment]
@TNgidi Validated in production, can close
